parm/atm/obs/config/lgetkf_amsua_n19.yaml

```diff
@@ -29,46 +29,46 @@ obs error:
 obs filters:
 - filter: Bounds Check
   filter variables:
-  - name: brightness_temperature
+  - name: brightnessTemperature
     channels: 4-6,9-14
   minvalue: 100.0
   maxvalue: 500.0
 - filter: Background Check
   filter variables:
-  - name: brightness_temperature
+  - name: brightnessTemperature
     channels: 4-6,9-14
   threshold: 3.0
 - filter: Domain Check
   filter variables:
-  - name: brightness_temperature
+  - name: brightnessTemperature
     channels: 4-6,9-14
   where:
   - variable:
-      name: scan_position@MetaData
+      name: sensorScanPosition@MetaData
     minvalue: 4
     maxvalue: 27
   - variable:
-      name: brightness_temperature_1@ObsValue
+      name: brightnessTemperature_1@ObsValue
```

Comment:
I thought we had got rid of the brightnessTemperature_<channum> format?

Also this should be ObsValue/brightnessTemperature, I think